authors/apps/articles/tests/test_create_article.py

```diff
@@ -0,0 +1,198 @@
+            "images": None
+        }
+        self.delete_article = {
+            "is_deleted": True
+        }
+
+    def signup_user_one(self):
+        """Function to register user one and return their token"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data,
+                                    format='json')
+        token = register.data['token']
+        return token
+
+    def signup_user_two(self):
+        """Function to register user two and return their token"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data2,
+                                    format='json')
+        token = register.data['token']
+        return token
+
+    def test_fetch_all_articles(self):
+        """Tests to fetch all articles"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        del self.create_article_data2['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data2,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.get(self.article_listcreate, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn(self.create_article_data['title'],
+                      response.data[0]['title'])
+        self.assertIn(self.create_article_data2['title'],
+                      response.data[1]['title'])
+        self.assertIn(self.create_article_data['body'],
+                      response.data[0]['body'])
+        self.assertIn(self.create_article_data2['body'],
+                      response.data[1]['body'])
+        self.assertIn(self.create_article_data['description'],
+                      response.data[0]['description'])
+        self.assertIn(self.create_article_data2['description'],
+                      response.data[1]['description'])
+
+    def test_add_articles(self):
+        """Test to add an article"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        response = self.client.post(self.article_listcreate,
+                                    self.create_article_data,
+                                    format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertIn(self.create_article_data['title'],
+                      response.data['title'])
+        self.assertIn(self.create_article_data['body'],
+                      response.data['body'])
+        self.assertIn(self.create_article_data['description'],
```

Comment:
`assertIn` should be used when the first argument is a subset of the second argument

Reply:
Done